tasks/src/table-question-answering/data.ts

```diff
@@ -0,0 +1,60 @@
+import type { Task DataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{ 
+			description:"The Fewshot Table dataset consists of tables that naturally occur on the web, that are formatted as few-shot tasks for fine-tuning language models to improve their few-shot performance.",
```

Comment:
Can we add a better documented canonical dataset, a couple including:
https://huggingface.co/datasets/wikisql
https://huggingface.co/datasets/msr_sqa
https://huggingface.co/datasets/wikitablequestions